person/views.py

```diff
@@ -35,34 +37,64 @@ class PersonDetail(DetailView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
+        # NOTE: this sort does not work as expected...
         last_cited_attr = '-object_ref__membershippersonlastciteddate'
-        memberships = context['person'].membershippersonmember_set\
-                        .order_by(last_cited_attr)
+        # memberships = context['person'].membershippersonmember_set\
+        #                 .order_by(last_cited_attr)
 
-        # Start by getting the most recent membership for the "Last seen as"
-        # description
+
+        # Use SQL...
+        membership_query = '''
+            SELECT * FROM membershipperson
+            WHERE member_id='{}' 
+            ORDER BY last_cited::date DESC
+        '''.format(context['person'].uuid)
+
+        cursor = connection.cursor()
+        cursor.execute(membership_query)
+
+        columns = [c[0] for c in cursor.description]
+        results_tuple = namedtuple('Membership', columns)
+
+        # List of membership tuples
+        memberships = [results_tuple(*r) for r in cursor]
+        
+
+        # Start by getting the most recent membership for the "Last seen as" description
         if len(memberships) > 0:
-            last_membership = memberships[0]
+            last_membership_id = memberships[0].id
+            membershipperson = MembershipPerson.objects.get(id=last_membership_id)
 
-            context['last_seen_as'] = last_membership.object_ref.short_description
+            context['last_seen_as'] = membershipperson.short_description
 
         context['memberships'] = []
         context['chain_of_command'] = []
```

Comment:
Do we use this `chain_of_command` array, or has it been fully superseded by `command_chain`?

Reply:
Nope. I'll delete it!